### ISSUE TITLE
2025-05-01, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#156]
# Today's Arxiv Papers

(1) [TRUST: An LLM-Based Dialogue System for Trauma Understanding and Structured Assessments](http://arxiv.org/abs/2504.21851v1), 2025-04-30

(2) [COMPACT: COMPositional Atomic-to-Complex Visual Capability Tuning](http://arxiv.org/abs/2504.21850v1), 2025-04-30

(3) [An Empirical Study on the Effectiveness of Large Language Models for Binary Code Understanding](http://arxiv.org/abs/2504.21803v1), 2025-04-30

(4) [DeepSeek-Prover-V2: Advancing Formal Mathematical Reasoning via Reinforcement Learning for Subgoal Decomposition](http://arxiv.org/abs/2504.21801v1), 2025-04-30

(5) [MAC-Tuning: LLM Multi-Compositional Problem Reasoning with Enhanced Knowledge Boundary Awareness](http://arxiv.org/abs/2504.21773v1), 2025-04-30

(6) [LASHED: LLMs And Static Hardware Analysis for Early Detection of RTL Bugs](http://arxiv.org/abs/2504.21770v1), 2025-04-30

(7) [LLM-based Interactive Imitation Learning for Robotic Manipulation](http://arxiv.org/abs/2504.21769v1), 2025-04-30

(8) [CodeFlowBench: A Multi-turn, Iterative Benchmark for Complex Code Generation](http://arxiv.org/abs/2504.21751v1), 2025-04-30

(9) [Investigating Literary Motifs in Ancient and Medieval Novels with Large Language Models](http://arxiv.org/abs/2504.21742v1), 2025-04-30

(10) [TheraQuest: A Gamified, LLM-Powered Simulation for Massage Therapy Training](http://arxiv.org/abs/2504.21735v1), 2025-04-30

(11) [LLM-Empowered Embodied Agent for Memory-Augmented Task Planning in Household Robotics](http://arxiv.org/abs/2504.21716v1), 2025-04-30

(12) [XBreaking: Explainable Artificial Intelligence for Jailbreaking LLMs](http://arxiv.org/abs/2504.21700v1), 2025-04-30

(13) [Visual Text Processing: A Comprehensive Review and Unified Evaluation](http://arxiv.org/abs/2504.21682v1), 2025-04-30

(14) [Hoist with His Own Petard: Inducing Guardrails to Facilitate Denial-of-Service Attacks on Retrieval-Augmented Generation of LLMs](http://arxiv.org/abs/2504.21680v1), 2025-04-30

(15) [Traceback of Poisoning Attacks to Retrieval-Augmented Generation](http://arxiv.org/abs/2504.21668v1), 2025-04-30

(16) [From Precision to Perception: User-Centred Evaluation of Keyword Extraction Algorithms for Internet-Scale Contextual Advertising](http://arxiv.org/abs/2504.21667v1), 2025-04-30

(17) [AdaR1: From Long-CoT to Hybrid-CoT via Bi-Level Adaptive Reasoning Optimization](http://arxiv.org/abs/2504.21659v1), 2025-04-30

(18) [Sadeed: Advancing Arabic Diacritization Through Small Language Model](http://arxiv.org/abs/2504.21635v1), 2025-04-30

(19) [Meeseeks: An Iterative Benchmark Evaluating LLMs Multi-Turn Instruction-Following Ability](http://arxiv.org/abs/2504.21625v1), 2025-04-30

(20) [RDF-Based Structured Quality Assessment Representation of Multilingual LLM Evaluations](http://arxiv.org/abs/2504.21605v1), 2025-04-30

(21) [Leveraging Pre-trained Large Language Models with Refined Prompting for Online Task and Motion Planning](http://arxiv.org/abs/2504.21596v1), 2025-04-30

(22) [DNB-AI-Project at SemEval-2025 Task 5: An LLM-Ensemble Approach for Automated Subject Indexing](http://arxiv.org/abs/2504.21589v1), 2025-04-30

(23) [MF-LLM: Simulating Collective Decision Dynamics via a Mean-Field Large Language Model Framework](http://arxiv.org/abs/2504.21582v1), 2025-04-30

(24) [Generative AI in Financial Institution: A Global Survey of Opportunities, Threats, and Regulation](http://arxiv.org/abs/2504.21574v1), 2025-04-30

(25) [Iterative Trajectory Exploration for Multimodal Agents](http://arxiv.org/abs/2504.21561v1), 2025-04-30

(26) [Precision Where It Matters: A Novel Spike Aware Mixed-Precision Quantization Strategy for LLaMA-based Language Models](http://arxiv.org/abs/2504.21553v1), 2025-04-30

(27) [Consistency-aware Fake Videos Detection on Short Video Platforms](http://arxiv.org/abs/2504.21495v1), 2025-04-30

(28) [Rethinking Visual Layer Selection in Multimodal LLMs](http://arxiv.org/abs/2504.21447v1), 2025-04-30

(29) [SeriesBench: A Benchmark for Narrative-Driven Drama Series Understanding](http://arxiv.org/abs/2504.21435v1), 2025-04-30

(30) [UAV-VLN: End-to-End Vision Language guided Navigation for UAVs](http://arxiv.org/abs/2504.21432v1), 2025-04-30

(31) [Who Gets the Callback? Generative AI and Gender Bias](http://arxiv.org/abs/2504.21400v1), 2025-04-30

(32) [In a Few Words: Comparing Weak Supervision and LLMs for Short Query Intent Classification](http://dx.doi.org/10.1145/3726302.3730213), 2025-04-30

(33) [Retrieval-Enhanced Few-Shot Prompting for Speech Event Extraction](http://arxiv.org/abs/2504.21372v1), 2025-04-30

(34) [ImaginateAR: AI-Assisted In-Situ Authoring in Augmented Reality](http://arxiv.org/abs/2504.21360v1), 2025-04-30

(35) [Nexus-Gen: A Unified Model for Image Understanding, Generation, and Editing](http://arxiv.org/abs/2504.21356v1), 2025-04-30

(36) [UniBiomed: A Universal Foundation Model for Grounded Biomedical Image Interpretation](http://arxiv.org/abs/2504.21336v1), 2025-04-30

(37) [Simple Visual Artifact Detection in Sora-Generated Videos](http://arxiv.org/abs/2504.21334v1), 2025-04-30

(38) [Does the Prompt-based Large Language Model Recognize Students' Demographics and Introduce Bias in Essay Scoring?](http://arxiv.org/abs/2504.21330v1), 2025-04-30

(39) [Covert Prompt Transmission for Secure Large Language Model Services](http://arxiv.org/abs/2504.21311v1), 2025-04-30

(40) [An Evaluation of a Visual Question Answering Strategy for Zero-shot Facial Expression Recognition in Still Images](http://arxiv.org/abs/2504.21309v1), 2025-04-30

(41) [Unsupervised Feature Transformation via In-context Generation, Generator-critic LLM Agents, and Duet-play Teaming](http://arxiv.org/abs/2504.21304v1), 2025-04-30

(42) [Confidence in Large Language Model Evaluation: A Bayesian Approach to Limited-Sample Challenges](http://arxiv.org/abs/2504.21303v1), 2025-04-30

(43) [BiasGuard: A Reasoning-enhanced Bias Detection Tool For Large Language Models](http://arxiv.org/abs/2504.21299v1), 2025-04-30

(44) [Birdie: Natural Language-Driven Table Discovery Using Differentiable Search Index](http://arxiv.org/abs/2504.21282v1), 2025-04-30

(45) [Reinforced MLLM: A Survey on RL-Based Reasoning in Multimodal Large Language Models](http://arxiv.org/abs/2504.21277v1), 2025-04-30

(46) [Assessing LLM code generation quality through path planning tasks](http://arxiv.org/abs/2504.21276v1), 2025-04-30

(47) [CoCoDiff: Diversifying Skeleton Action Features via Coarse-Fine Text-Co-Guided Latent Diffusion](http://arxiv.org/abs/2504.21266v1), 2025-04-30

(48) [Talk Before You Retrieve: Agent-Led Discussions for Better RAG in Medical QA](http://arxiv.org/abs/2504.21252v1), 2025-04-30

(49) [Memorization and Knowledge Injection in Gated LLMs](http://arxiv.org/abs/2504.21239v1), 2025-04-30

(50) [Phi-4-Mini-Reasoning: Exploring the Limits of Small Reasoning Language Models in Math](http://arxiv.org/abs/2504.21233v1), 2025-04-30

(51) [CachePrune: Neural-Based Attribution Defense Against Indirect Prompt Injection Attacks](http://arxiv.org/abs/2504.21228v1), 2025-04-29

(52) [Theoretical Foundations for Semantic Cognition in Artificial Intelligence](http://arxiv.org/abs/2504.21218v1), 2025-04-29

(53) [A Cost-Effective LLM-based Approach to Identify Wildlife Trafficking in Online Marketplaces](http://dx.doi.org/10.1145/3725256), 2025-04-29

(54) [SecRepoBench: Benchmarking LLMs for Secure Code Generation in Real-World Repositories](http://arxiv.org/abs/2504.21205v1), 2025-04-29

(55) [Automatic Legal Writing Evaluation of LLMs](http://arxiv.org/abs/2504.21202v1), 2025-04-29

(56) [Graph Synthetic Out-of-Distribution Exposure with Large Language Models](http://arxiv.org/abs/2504.21198v1), 2025-04-29

(57) [Small or Large? Zero-Shot or Finetuned? Guiding Language Model Choice for Specialized Applications in Healthcare](http://arxiv.org/abs/2504.21191v1), 2025-04-29

(58) [LIFT: LLM-Based Pragma Insertion for HLS via GNN Supervised Fine-Tuning](http://arxiv.org/abs/2504.21187v1), 2025-04-29

(59) [GLIP-OOD: Zero-Shot Graph OOD Detection with Foundation Model](http://arxiv.org/abs/2504.21186v1), 2025-04-29

(60) [AI-in-the-Loop Planning for Transportation Electrification: Case Studies from Austin, Texas](http://arxiv.org/abs/2504.21185v1), 2025-04-29

(61) [Efficient LLMs with AMP: Attention Heads and MLP Pruning](http://arxiv.org/abs/2504.21174v1), 2025-04-29

(62) [Detecting Manipulated Contents Using Knowledge-Grounded Inference](http://arxiv.org/abs/2504.21165v1), 2025-04-29

(63) [LLM Enhancer: Merged Approach using Vector Embedding for Reducing Large Language Model Hallucinations with External Knowledge](http://arxiv.org/abs/2504.21132v1), 2025-04-29

(64) [Beyond One-Size-Fits-All: Inversion Learning for Highly Effective NLG Evaluation Prompts](http://arxiv.org/abs/2504.21117v1), 2025-04-29

(65) [Toward Efficient Exploration by Large Language Model Agents](http://arxiv.org/abs/2504.20997v1), 2025-04-29

(66) [X-Fusion: Introducing New Modality to Frozen Large Language Models](http://arxiv.org/abs/2504.20996v1), 2025-04-29

(67) [ACE: A Security Architecture for LLM-Integrated App Systems](http://arxiv.org/abs/2504.20984v1), 2025-04-29

(68) [On the Potential of Large Language Models to Solve Semantics-Aware Process Mining Tasks](http://arxiv.org/abs/2504.21074v1), 2025-04-29

(69) [Jekyll-and-Hyde Tipping Point in an AI's Behavior](http://arxiv.org/abs/2504.20980v1), 2025-04-29

(70) [Real-Time Wayfinding Assistant for Blind and Low-Vision Users](http://arxiv.org/abs/2504.20976v1), 2025-04-29

(71) [SetKE: Knowledge Editing for Knowledge Elements Overlap](http://arxiv.org/abs/2504.20972v1), 2025-04-29

(72) [AegisLLM: Scaling Agentic Systems for Self-Reflective Defense in LLM Security](http://arxiv.org/abs/2504.20965v1), 2025-04-29

(73) [OSVBench: Benchmarking LLMs on Specification Generation Tasks for Operating System Verification](http://arxiv.org/abs/2504.20964v1), 2025-04-29

(74) [Information Gravity: A Field-Theoretic Model for Token Selection in Large Language Models](http://dx.doi.org/10.5281/zenodo.15289890), 2025-04-29

(75) [Trace-of-Thought: Enhanced Arithmetic Problem Solving via Reasoning Distillation From Large to Small Language Models](http://arxiv.org/abs/2504.20946v1), 2025-04-29

(76) [ChestX-Reasoner: Advancing Radiology Foundation Models with Reasoning through Step-by-Step Verification](http://arxiv.org/abs/2504.20930v1), 2025-04-29

(77) [DYNAMAX: Dynamic computing for Transformers and Mamba based architectures](http://arxiv.org/abs/2504.20922v1), 2025-04-29

(78) [An Empirical Study on the Capability of LLMs in Decomposing Bug Reports](http://arxiv.org/abs/2504.20911v1), 2025-04-29

(79) [Classifier-to-Bias: Toward Unsupervised Automatic Bias Detection for Visual Classifiers](http://arxiv.org/abs/2504.20902v1), 2025-04-29

(80) [A Systematic Literature Review of Parameter-Efficient Fine-Tuning for Large Code Models](http://arxiv.org/abs/2504.21569v1), 2025-04-29

(81) [LELANTE: LEveraging LLM for Automated ANdroid TEsting](http://arxiv.org/abs/2504.20896v1), 2025-04-29

(82) [The Leaderboard Illusion](http://arxiv.org/abs/2504.20879v1), 2025-04-29

(83) [JaccDiv: A Metric and Benchmark for Quantifying Diversity of Generated Marketing Text in the Music Industry](http://arxiv.org/abs/2504.20849v1), 2025-04-29

(84) [Enhancing Non-Core Language Instruction-Following in Speech LLMs via Semi-Implicit Cross-Lingual CoT Reasoning](http://arxiv.org/abs/2504.20835v1), 2025-04-29

(85) [Reinforcement Learning for LLM Reasoning Under Memory Constraints](http://arxiv.org/abs/2504.20834v1), 2025-04-29

(86) [Ascendra: Dynamic Request Prioritization for Efficient LLM Serving](http://arxiv.org/abs/2504.20828v2), 2025-04-29

(87) [Secure Coding with AI, From Creation to Inspection](http://arxiv.org/abs/2504.20814v1), 2025-04-29

(88) [Unlocking User-oriented Pages: Intention-driven Black-box Scanner for Real-world Web Applications](http://arxiv.org/abs/2504.20801v2), 2025-04-29

(89) [Hallucination by Code Generation LLMs: Taxonomy, Benchmarks, Mitigation, and Challenges](http://arxiv.org/abs/2504.20799v1), 2025-04-29

(90) [Q-Fusion: Diffusing Quantum Circuits](http://arxiv.org/abs/2504.20794v1), 2025-04-29

(91) [Using LLMs in Generating Design Rationale for Software Architecture Decisions](http://arxiv.org/abs/2504.20781v1), 2025-04-29

(92) [Turing Machine Evaluation for Large Language Model](http://arxiv.org/abs/2504.20771v1), 2025-04-29

(93) [Chain-of-Defensive-Thought: Structured Reasoning Elicits Robustness in Large Language Models against Reference Corruption](http://arxiv.org/abs/2504.20769v1), 2025-04-29

(94) [Understanding Large Language Model Supply Chain: Structure, Domain, and Vulnerabilities](http://arxiv.org/abs/2504.20763v1), 2025-04-29

(95) [Beyond the Last Answer: Your Reasoning Trace Uncovers More than You Think](http://arxiv.org/abs/2504.20708v1), 2025-04-29

(96) [Can LLMs Detect Intrinsic Hallucinations in Paraphrasing and Machine Translation?](http://arxiv.org/abs/2504.20699v1), 2025-04-29

(97) [Identifying Uncertainty in Self-Adaptive Robotics with Large Language Models](http://arxiv.org/abs/2504.20684v1), 2025-04-29

(98) [CoCo-Bench: A Comprehensive Code Benchmark For Multi-task Large Language Model Evaluation](http://arxiv.org/abs/2504.20673v1), 2025-04-29

(99) [A Generative-AI-Driven Claim Retrieval System Capable of Detecting and Retrieving Claims from Social Media Platforms in Multiple Languages](http://arxiv.org/abs/2504.20668v1), 2025-04-29

(100) [ComplexVCoder: An LLM-Driven Framework for Systematic Generation of Complex Verilog Code](http://arxiv.org/abs/2504.20653v1), 2025-04-29
